spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -251,7 +251,10 @@ def fixture_record(fixture_name)
       expect(@record_call_number2['call_number_browse_s']).to eq(["CD- 50000"])
     end
     it 'returns the enrichment call_number_browse with k at the end' do
-      expect(@sample41['call_number_browse_s']).to eq(["GA 2015.00160 A Middle 30/Drawer 11/GC024/Full Folio/20th-21st c./Artists A"])
+      expect(@sample41['call_number_browse_s']).to contain_exactly("GA 2015.00160", "GA 2015.00160 A Middle 30/Drawer 11/GC024/Full Folio/20th-21st c./Artists A")
```

Comment:
@christinach I believe the `GA 2015.00160` being repeated here is unexpected: https://catalog.princeton.edu/catalog/99106471643506421 There is only one item.  It does not have an 8 field, but neither does the SCSB record.